### PR TITLE
chore(build): allow running linked modules

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

File: config/webpack.common.js [hunk 1]
```diff
@@ -125,7 +125,7 @@ module.exports = function (options) {
        *
        * See: https://webpack.js.org/configuration/resolve/#resolve-modules
        */
-      // modules: [ path.resolve(__dirname, 'src'), path.resolve(__dirname, 'node_modules')]
+      modules: [ helpers.root('src'), helpers.root('node_modules')]
```
Comment (joshuawilson) [inline config/webpack.common.js]: I get the error that modules is not allowed. That is why I commented it out.

Comment (pmuir) [inline config/webpack.common.js]: You mean that the configuration itself option is not allowed? Double check this one, as this solved the link problem for me.

Comment (joshuawilson) [inline config/webpack.common.js]: The problem is that the tests fail. The app will start but we can't test it.

Comment (pmuir) [inline config/webpack.common.js]: So let's run with this in dev but not in test :-) I'll set that up, as this really does work for me.

Comment (pmuir) [inline config/webpack.common.js]: BTW I don't understand the comment the tests fail - CI is passing, the unit tests pass for me locally...

**Comment by pmuir** [comment]
[test]

**Comment by pmuir** [comment]
@joshuawilson tests passing on CI - please shout if you still have problems.

**Comment by joshuawilson** [comment]
it still fails for me locally but I'll merge it and then take a look again

**Pull request merged**